MovieApp/LoginViewController.swift

```diff
@@ -0,0 +1,217 @@
+import Foundation
+import UIKit
+import PureLayout
+
+// https://stackoverflow.com/questions/25367502/create-space-at-the-beginning-of-a-uitextfield
+class PaddedTextField: UITextField {
+
+    let padding = UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 16)
+
+    override open func textRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+
+    override open func placeholderRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+
+    override open func editingRect(forBounds bounds: CGRect) -> CGRect {
+        return bounds.inset(by: padding)
+    }
+}
+
+class LogInViewController: UIViewController{
+    
+    private let viewBackgroundColor = UIColor(red: 19/255, green: 59/255, blue: 99/255, alpha: 1)
```

Comment:
You can create a separate file to store colors. Can make an extension on UIColor and use static properties. Google it a bit.